eventlet/green/subprocess.py

```diff
@@ -103,17 +101,14 @@ def wait(self, timeout=None, check_interval=0.01):
         # just want a version that uses eventlet.green.select.select()
         # instead of select.select().
         _communicate = FunctionType(
-            six.get_function_code(six.get_unbound_function(
-                subprocess_orig.Popen._communicate)),
+            subprocess_orig.Popen._communicate.__code__,
             globals())
         try:
             _communicate_with_select = FunctionType(
-                six.get_function_code(six.get_unbound_function(
```

Comment:
Not sure to see why these calls were duplicated... doesn't matter now that they are removed